lib/workers/repository/update/pr/index.ts

```diff
@@ -82,16 +89,34 @@ export async function ensurePr(
   );
 
   const config: BranchConfig = { ...prConfig };
-
+  const prFingerprint = fingerprint(config);
   logger.trace({ config }, 'ensurePr');
   // If there is a group, it will use the config of the first upgrade in the array
   const { branchName, ignoreTests, prTitle = '', upgrades } = config;
   const dependencyDashboardCheck =
     config.dependencyDashboardChecks?.[config.branchName];
-  // Check if existing PR exists
+  // Check if PR already exists
   const existingPr = await platform.getBranchPr(branchName);
   if (existingPr) {
     logger.debug('Found existing PR');
+    const prCache = getPrCache(branchName);
+    if (prCache) {
+      logger.debug({ prCache }, 'Found existing Pr cache');
+      const lastEditTime = prCache.lastEdited;
+      // return if pr cache is valid and pr was last edited before a day 
+      if (
+        isCloned() === false &&
```

Comment:
Why check isCloned?

Reply:
I began working with the idea that we should only try to avoid PR updates if the branch update has been skipped.
However, there may be some circumstances when we update the branch but do not need to update the PR like any update that is not done because of a upgrades difference. Will revert this one and re-run on a real repo